src/lib/y2network/autoinst_profile/interface_section.rb

```diff
@@ -259,6 +259,9 @@ def self.new_from_network(connection_config)
       def initialize(*_args)
         super
 
+        # TODO: Initializing all the attributes to an empty string makes
+        # hard to know whether the value was defined or not at all. We probably
+        # should ommit this initialization
```

Comment:
well, on other hand we should not have nil at all. So it is better then state with either nil or "" everywhere. At least that was idea.